go.sum

```diff
@@ -1,21 +1,300 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
cobra has massive deps. personally I've switched to urfave, but didn't want to raise something as dramatic as first PR plus no idea your pref :D